modules/fts/pages/fts-troubleshooting.adoc

```diff
@@ -160,3 +160,52 @@ These are scorch specifc commands ...
  ** command-line tool to interact with a zap file
 
 Invoking the commands above with --help will highlight more information and further sub commands available for each.
+
+*Q: How does FTS scores documents?*
+
+FTS's internal text indexing library(bleve) uses a slightly modified version of standard tf-idf scoring. This improvisation is done to normalise the score by various relevant factors. The search scoring happens at the query time.
+
+When bleve scores a document - it sort of sums a set of sub scores to reach the final score. Scores across different searches are not directly comparable as the search query is also an input factor to the scoring function. The more conjuncts/disjuncts/sub clauses your query has, the more it will influence the scoring.
+The score of a particular hit is not absolute, meaning that it can only be used as a comparison to the highest score from the same search result. There isn't a pre-defined range for valid scores. 
+
+To summarise the scoring function in a comprehensive way,
+
+Given a document which has a field  `f`  over which a given  match query `q`  is applied, 
+then the  `scoreFn`  for that document is defined as:
+
+----
+scoreFn(q, f) = coord(q, f) * SUM(tw(t0, q, f), tw(t1, q, f), tw(t2, q, f)..., tw(tn, q, f))
+where ti := term in q
+coord(q, f) = nFoundTokens(q, f)/nTokens(q)
+tw(ti, q, f) = queryWeight(q, f, ti) * fieldWeight(f, ti)
+queryWeight(q, ti) = w(ti) * queryNorm(q)
+w(ti) = boost(ti) * idf(ti)
+queryNorm(q) = 1 / SQROOT(SUM(SQ(w(t0)),...,SQ(w(tn))))
+fieldWeight(f, ti) = SQROOT(FREQ(ti, f))*idf(f, ti)*fieldNorm(f)
+fieldNorm(f) = 1 / SQROOT(nTokens(f))
+idf(f, ti) = 1 + LN(|Docs| / (1 + FREQ(ti, FIELDNAME(f), Docs)))
+Docs = a set of all indexed documents
+----
+
+where SQROOT, SUM, and LN denote standard mathematical functions. Auxiliary functions are:
+
+* *coord(q, f)*  — is a dampening factor defined as a ratio of query tokens that are found in the given field, and the total number of tokens in a query.
+* *tw(ti, q, f)*  — *ti* ’s term weight is the product of  *ti* ’s query weight and ti’s field weight.
+* *queryWeight(q, ti)*  —  *ti* ’s query weight (wrt to  *q* ) is the product of its [inverse document frequency](https://en.wikipedia.org/wiki/Tf%E2%80%93idf) (see  *idf*  below) and its [boosting factor](http://www.blevesearch.com/docs/Query-String-Query/).
```

Comment:
INFO: External links are in the following format: URL[Link text to display] 
For example, here are a few that need updating. Please update all the other links too.
- https://en.wikipedia.org/wiki/Tf%E2%80%93idf[inverse document frequency]
- http://www.blevesearch.com/docs/Query-String-Query/[boosting factor]